README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# HOMEWORK ONE   
+## Who is Paulino?
+![](https://github.com/pdiazm/hw01/blob/master/PD_Headshot.jpg)
+
+### __Background__:
+Paulino is a communications consultant and Harris School of Public Policy MPP candidate, passionate about the intersection of policy and cyber-technology. He is looking for ways to empower society through a more open and transparent infrastructure for the accumulation, consumption and dissemination of information. Before joining Harris, Paulino worked as a media relations manager at global public relations agency Golin, were he helped develop, coordinate and implement multi-faceted communications and public relations campaigns, at the local, national and international level.
+
+### __Basic Info__:   
+* _Name_: Paulino Diaz Mejia   
```

Comment:
As the "basic information" contains information such as full name and organisational affiliation, it could have preceded the short "background" paragraph.